tests/unit/schema/wrappers/test_method.py

```diff
@@ -189,6 +190,61 @@ def test_method_paged_result_field_no_page_field():
     assert method.paged_result_field is None
 
 
+def test_method_paged_result_field_invalid_wrapper_type():
+    """Validate paged_result_field() returns None if page_size/max_results wrappertypes
+    are not allowed types.
+    """
+
+    # page_size is not allowed wrappertype
+    parent = make_field(name="parent", type=9)  # str
```

Comment:
```suggestion
    parent = make_field(name="parent", type="TYPE_STRING")
```

Use the enum string rather than the numeric value as it improves readability similar to https://github.com/googleapis/gapic-generator-python/blob/c8b7229ba2865d6a2f5966aa151be121de81f92d/tests/unit/schema/wrappers/test_field.py#L275

You can find all of the types here:
https://github.com/protocolbuffers/protobuf/blob/836fb77aebc3bc4a6e2508b67e4ab7977da3bcc1/src/google/protobuf/descriptor.proto#L241

Applies throughout

Reply:
@parthea

Resolved.